core-bundle/src/Resources/contao/themes/flexible/main.css

```diff
@@ -2139,6 +2139,73 @@ fieldset.collapsed > legend {
 	background-color: #f7f7f9 !important;
 }
 
+/* UNDO */
+.listing__tl_undo.list_view .tl_file_list {
+	border-top: 1px solid darkgray;
+}
+.listing__tl_undo.list_view .tl_folder_tlist,
+.listing__tl_undo.list_view .tl_folder_list {
+	background-color: #eaeaec;
+}
+.listing__tl_undo.list_view .tl_listing tr.odd td {
+	background-color: #f4f4f4;
+}
+.listing__tl_undo.list_view .tl_file_list {
+	border-bottom: 1px solid darkgray;
+}
+.listing__tl_undo.list_view tr:last-child .tl_file_list {
+	border-bottom: 0;
+}
+.listing__tl_undo.list_view .tl_file_list:first-child {
+	padding: 0!important;
+}
+.listing__tl_undo.list_view .tl_file_list:first-child > div:nth-child(2) {
+	padding: 16px 16px 16px 6px !important;
+}
+.listing__tl_undo.list_view .tl_file_list:nth-child(2) {
+	border-left: 1px solid #e9e9e9;
+	padding: 0 8px!important;
+	background-color: #f4f4f4;
+}
+.tl_undo_header {
+	display: flex;
+	padding: 7px 6px;
+	border-bottom: 1px solid #e9e9e9;
+}
+.hover-row:hover .tl_undo_header {
+	background-color: #fffce1!important;
+	border-bottom-color: #fffce1;
+}
+.hover-row:hover .tl_file_list:nth-child(2) {
+	border-left-color: #fffce1;
+}
+.listing__tl_undo.list_view .odd .limit_toggler {
+	background-color: #fafafc;
+}
+.tl_undo_header .tl_undo_header_item:not(:first-child) {
+	margin-left: 32px;
+}
+.tl_undo_header_item.align-right {
+	align-self: right;
```

Comment:
<img width="897" alt="" src="https://user-images.githubusercontent.com/1192057/149291107-cf0b29d5-b669-4910-85b1-6f6add92797c.png">

Reply:
Seems to be a left-over. I removed it in 65e8a177dfae88315905dbc83adbee5cfb396961